packages/jsii/bin/jsii.ts

```diff
@@ -41,7 +40,7 @@ const warningTypes = Object.keys(enabledWarnings);
           .option('fix-peer-dependencies', {
             type: 'boolean',
             default: true,
-            desc: 'Automatically add missing entries in the peerDependencies section of package.json',
+            desc: 'Automatically add missing entries in the peerDependencies section of package.json (deprecated, defunct)',
```

Comment:
I'd say we hide this from the output of `jsii --help` and make it clear that it is now NOOP.

```suggestion
            desc: 'This option no longer has any effect.',
            hidden: true,
```